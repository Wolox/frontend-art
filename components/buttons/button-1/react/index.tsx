import React from 'react';

import '../styles.scss';

function Button1() {
  return (
    <div className="column center btn-group-1">
      <button className="btn btn-1 m-bottom-2" disabled={true}>
        <div className="btn__content">cancel</div>
      </button>
      <button type="button" className="btn btn-1">
        <div className="btn__content">submit</div>
      </button>
    </div>
  );
}

export default Button1;
