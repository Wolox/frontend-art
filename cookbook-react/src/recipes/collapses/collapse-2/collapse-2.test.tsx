import React from 'react';
import { render } from '@testing-library/react';

import Collapse from './index';

const title = 'Título / Pregunta / Ítem';
const text =
  'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.';

test('renders correctly', () => {
  const { container } = render(<Collapse title={title} text={text} />);
  expect(container).toMatchSnapshot();
});
