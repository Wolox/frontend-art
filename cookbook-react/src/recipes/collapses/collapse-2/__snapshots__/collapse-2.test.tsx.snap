// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="collapseWrapper"
  >
    <div
      class="titleContainer"
    >
      <h2
        class="title"
      >
        Título / Pregunta / Ítem
      </h2>
      <button
        class="arrow up"
        type="button"
      />
    </div>
    <div
      class="textContainer"
    >
      <p
        class="text"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </p>
    </div>
  </div>
</div>
`;
